packages/packages/documents/src/sync/__tests__/sync-store.test.ts

```diff
@@ -0,0 +1,259 @@
+import { createSlice } from '../../store';
+import { syncStore } from '../sync-store';
+import { ExtendedWindow, Slice, V1Document } from '../../types';
+import { flushListeners } from '@elementor/v1-adapters';
+import { createStore, deleteStore, SliceState, Store } from '@elementor/store';
+import {
+	dispatchCommandAfter,
+	dispatchCommandBefore,
+	dispatchV1ReadyEvent,
+	makeDocumentsManager,
+	makeMockV1Document,
+} from './test-utils';
+
+type WindowWithV1Loading = Omit<ExtendedWindow, 'elementor'> & {
+	__elementorEditorV1LoadingPromise?: Promise<void>;
+	elementor?: ExtendedWindow['elementor'];
+}
+
+describe( '@elementor/documents/sync/store-sync', () => {
+	let store: Store<SliceState<Slice>>;
+	let slice: Slice;
+	let extendedWindow: WindowWithV1Loading;
+
+	beforeEach( () => {
+		slice = createSlice();
+		store = createStore();
+
+		syncStore( slice );
+
+		extendedWindow = ( window as unknown as WindowWithV1Loading );
+		extendedWindow.__elementorEditorV1LoadingPromise = Promise.resolve();
```

Comment:
can you just trigger the event of the init by yourself?
seems like this area knows too much implementation details

Reply:
It's actually unnecessary now... removed 😄